Stratis.Bitcoin.Features.Miner/PosMinting.cs

```diff
@@ -215,7 +215,7 @@ public void GenerateBlocks(WalletSecret walletSecret)
                     // unless we are completely disconnected. So this is weird logic.
                     bool fewPeers = this.connection.ConnectedNodes.Count() < 3;
                     bool lastBlockTooOld = this.chain.Tip.Header.Time < (this.dateTimeProvider.GetTime() - 10 * 60);
-                    if (fewPeers || lastBlockTooOld)
+                    if ((fewPeers && !this.network.IsTest()) || lastBlockTooOld)
```

Comment:
I dont think we need this. take it away now and we can discuss later.

Reply:
This causes problems when developing on testnet. And I'm not sure how to make a partial change in file that I would avoid being committed to the repository. Will discuss on slack